VERSION_NUMBER

```diff
@@ -1 +1 @@
-1.4.0
+1.4.1
```

Comment:
This should be changed by mistake?

Reply:
I have not changed this, only merged the master